app/constants/events.js

```diff
@@ -0,0 +1,15 @@
+module.exports = {
+  PAYMENT_EXTRACTED: 'uk.gov.defra.ffc.pay.payment.extracted',
+  PAYMENT_ENRICHED: 'uk.gov.defra.ffc.pay.payment.enriched',
+  PAYMENT_PAUSED_DEBT: 'uk.gov.defra.ffc.pay.payment.paused.debt',
+  PAYMENT_DEBT_ATTACHED: 'uk.gov.defra.ffc.pay.payment.debt.attached',
+  PAYMENT_PAUSED_LEDGER: 'uk.gov.defra.ffc.pay.payment.paused.ledger',
+  PAYMENT_LEDGER_ASSIGNED: 'uk.gov.defra.ffc.pay.payment.ledger.assigned',
+  PAYMENT_PAUSED_QUALITY_CHECK: 'uk.gov.defra.ffc.pay.payment.ledger.quality-check.pending',
+  PAYMENT_FAILED_QUALITY_CHECK: 'uk.gov.defra.ffc.pay.payment.ledger.quality-check.failed',
+  PAYMENT_PASSED_QUALITY_CHECK: 'uk.gov.defra.ffc.pay.payment.ledger.quality-check.passed',
```

Comment:
Would it be better to keep these variable names the same flow as the others eg. PAYMENT_QUALITY_CHECK_PENDING (if we're wanting to omit 'ledger' here), or do you think the way you've got these now is fine?

Reply:
Agree about, passed and failed, have switched them to be consistent with ledger.

Paused is a bit different as I view paused as the main activity and either ledger or quality as the reason.

Reply:
Yep, that makes sense